enterprise_gateway/services/processproxies/k8s.py

```diff
@@ -308,14 +312,61 @@ def _create_kernel_namespace(self, service_account_name: str) -> str:
                     body = client.V1DeleteOptions(
                         grace_period_seconds=0, propagation_policy="Background"
                     )
-                    client.CoreV1Api().delete_namespace(name=namespace, body=body)
+                    client.CoreV1Api(api_client=kubernetes_client).delete_namespace(name=namespace, body=body)
                     self.log.warning(f"Deleted kernel namespace: {namespace}")
                 else:
                     reason = f"Error occurred creating namespace '{namespace}': {err}"
                 self.log_and_raise(http_status_code=500, reason=reason)
 
         return namespace
 
+    def _create_service_account_if_not_exists(self, namespace: str, service_account_name: str) -> None:
+        service_account_list_in_namespace: client.V1ServiceAccountList = client\
+            .CoreV1Api(api_client=kubernetes_client)\
+            .list_namespaced_service_account(namespace=namespace)
```

Comment:
(More of a meta point than a specific set of lines)

Slash continuations in Python are pretty evil.  Was this something you put in manually, or something the formatter did for you?

You might want to put these on one (really long) line, and then use `black` to find a solution that doesn't involve slashes. I believe the project uses `black` anyway, so it is a good idea to run that over your code from the getgo.

Reply:
PyCharm did it when I tried to chain the invocations like I usually do in js/ts, honestly wasn't aware slash continuations were a problem until I just googled it, so thanks for pointing this one out